backend/controller/observability/async_calls.go

```diff
@@ -132,14 +134,19 @@ func (m *AsyncCallMetrics) Completed(ctx context.Context, verb schema.RefKey, or
 	m.queueDepth.Record(ctx, queueDepth)
 }
 
-func extractAsyncCallAttrs(verb schema.RefKey, origin string, scheduledAt time.Time) []attribute.KeyValue {
-	return append(extractRefAttrs(verb, origin), attribute.String(asyncCallTimeSinceScheduledAtBucketAttr, logBucket(8, timeSinceMS(scheduledAt))))
+func extractAsyncCallAttrs(verb schema.RefKey, caughtByVerb optional.Option[schema.RefKey], origin string, scheduledAt time.Time, isCatching bool) []attribute.KeyValue {
```

Comment:
nit: s/caughtByVerb/catchVerb/ since this function will get hit even when we're recording an async call event that is _not_ a catch (e.g. we're just creating an async call that has a catch verb)

Reply:
🤦‍♂️ missed that one when I renamed things earlier, thanks